framework/src/auxkernels/ElementH1ErrorFunctionAux.C

```diff
@@ -23,6 +23,9 @@ InputParameters
 validParams<ElementH1ErrorFunctionAux>()
 {
   InputParameters params = validParams<ElementL2ErrorFunctionAux>();
+  params.addClassDescription(
+      "Computes the H1 or W^{1,p} error between an exact function and a coupled variable");
```

Comment:
Add period at end. Ideally, each of the class descriptions should be a complete sentence, that way when they are included on the page the read correctly.